### DISCUSSION TITLE
Command Line Agent install for Windows

### DISCUSSION THREAD

**Discussion by ghostersk** [Ylianst/MeshCentral#4634] [Ideas]
I found the terminal installer for Linux super easy.
Is there some command line command how to install/manage the Windows agent?
if it is not, would this be possible another feature to add?
Thank you

**Comment by dinger1986** [comment]
You can install the windows agent command line with meshagent.exe -fullinstall uninstall with -fulluninstall

**Reply by ghostersk** [reply]
Thank you, I did not realized it.
it works nicely.
Now I know how to download the file and run it from powershell.

iwr -URI 'the link to windows executable' -Outfile mesh.exe
.\mesh.exe -run

**Comment by noagenda33** [comment]
Is there a command to install to a specific group? This way I could use a single .EXE for all groups.

**Comment by gasmer5556** [comment]
The line that is used from Remotely is like this:
C:\Support\Remotely_Installer.exe -install -quiet -organizationid "codehere" -serverurl "http://xxx.xxx.xxx.xxx:5000" -devicegroup "Laptop PCS"

Would it be similar?

**Reply by dinger1986** [reply]
Just need to download the agent from your mesh server and use meshagent -fullinstall
